apps/src/constants.js

```diff
@@ -57,6 +57,7 @@ exports.TestResults = {
   LOCAL_FUNCTION_FAIL: -6,            // The program contains an unexpected JavaScript local function
   GENERIC_LINT_FAIL: -7,              // The program contains a lint error
   LOG_CONDITION_FAIL: -8,             // The program execution log did not pass a required condition
+  SKIPPED: -9,                        // Skipped, e.g. they used the skip button on a challenge level
```

Comment:
Given that we aggregate `UserLevel.best_result` to be the maximum, I'd think we might want to make this `-100` or some such (should it be greater than or less than `UNSUBMITTED_ATTEMPT`) to allow additional values greater than it.